lib/cli/src/cli.rs

```diff
@@ -364,6 +384,13 @@ impl FromStr for SplitVersion {
 
         let mut no_version = false;
 
+        if s.starts_with('/') || s.contains("..") {
```

Comment:
`./test.wasm` is a valid path.
As is `hello/./test.wasm`.

This doesn't seem to be handled.
But I don't know the wider implications for the rest of the code.